README.md

```diff
@@ -20,10 +20,20 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   cd ..
   catkin_make
   ```
+  
+### Application can work at two modes:
+-  Local mode
+  -  All the ROS nodes run at the same machine.
+-  Network mode
+  -  ROS nodes run at two different machine: Server and Client.
+  -  Client runs ROS nodes "RVIZ" and "point_cloud_generator_node".
+  -  Server runs all other ROS nodes except two nodes running on client.
```

Comment:
About the clock sync, I did some try by adding one ROS node "Clock_server" at server side to publish topic /clock, and other ROS nodes use time from /clock instead of wall-clock time.

 I encounter two problems so far:
  1). Can't start point_cloud_generator_node, because of runtime_error, 
after debugging I think the root cause is that  ros::Time::now() will return 0 before it receive the topic /clock,  then msg.header.stamp = ros::Time::now() - loop_duration return negative causing the problem, I add condition only when  ros::Time::now() > 0 to continue, now point_cloud_generator_node can start now.

Errlog:
> 
> terminate called after throwing an instance of 'std::runtime_error'
>   what():  Time is out of dual 32-bit range
> [ INFO] [1461016385.817689728]: rviz version 1.11.10
> [ INFO] [1461016385.817749097]: compiled against OGRE version 1.8.1 (Byatis)
> [opnear4/point_cloud_generator_node-1] process has died [pid 15124, exit code -6, cmd 
 
  2). Hit Scan button, the simulated robot didn't move as we expected. 
 I didn't figure out what the root cause yet.

Then I read the link about [clock](http://wiki.ros.org/Clock), it recommended that:
>When using wall-clock time on multiple machines, it is important that you synchronize time between them. ROS does not provide this functionality as there are already well-established methods (e.g. ntp, our recommended syncronization tool is chrony) for doing this

I tried to use chrony to test two scenarios:
1. Server and client at same timezone, after time synchronized, the blending system works fine.
2. Server and client at different timezone, after time synchronized, the blending system works fine.

Since now we need to work at several scenarios for  Austin Demo, for the clock synchronize issue, can we use the NTP/Chrony time for now? After the most work done, we can go back to try again and see  if we can get the simulate clock work.